code/game/objects/items/robot/robot_items.dm

```diff
@@ -975,8 +976,7 @@
 	/obj/item/reagent_containers/glass/bucket
 	)
 
-/obj/item/borg/apparatus/beaker/service/examine()
+/obj/item/borg/apparatus/beaker/service/Initialize()
 	. = ..()
-	if(stored)
-		. += "You are currently holding [stored]."
-		. += "<span class='notice'<i>Alt-click</i> will drop the currently stored [stored].</span>"
+	stored = new /obj/item/reagent_containers/food/drinks/drinkingglass(src)
+	update_icon()
```

Comment:
Why are you removing the examine message? Is that already a part of beaker?

Reply:
Yes it is, and currently this results in the same line displaying twice in one examine